workshop-resources/keyboard-navigation/event-reviews/2022-10-12-notebook7.md

```diff
@@ -0,0 +1,210 @@
+# Notebook 7 - Collaborative keyboard navigation review
+
+Please add your findings as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) so you are credited in the commit history!
+
+## Keyboard navigation accessibility review of [tool name]
+
+Reviewed via based on [this link we'll all be working off of]().
+
+- Review date:
+- Operating system(s):
+- Browser(s): 
+- Keyboard language(s) used:
+- Any other details (ie. dependencies, github repo, etc.):
+
+### Authors
+
+Participants may add their name below if they would like to be listed publicly as a part of the review.
+
+1.
+2.
+3.
+4.
+5. 
+6.
+7.
+8.
+9.
+10. 
+
+## Review
```

Comment:
Review might be better split by view (ie. what is open, where) or UI region rather than test running over all of it.